dev-docs/conventions.md

```diff
@@ -171,11 +171,13 @@ and `<Title>` starts with a capital letter.
 
 ## CLI reference
 
-The command reference within the CLI should follow the [Cobra](https://github.com/spf13/cobra) style which features the following:
+The command reference within the CLI should follow the following conventions:
 
-- Short description: Starts with a capital letter, beginnings of sentences, names and acronyms are capitalized, ends without a period.
+- Short description: Starts with a capital letter, beginnings of sentences, names and acronyms are capitalized, ends without a period. It should be a single sentence.
```

Comment:
Maybe short descriptions should be formatted like flags?

Reply:
Since they are displayed in the same way in the Cobra CLI, they probably should. I will change this